layouts/partials/cookie_consent.html

```diff
@@ -0,0 +1,5 @@
+<script type="text/javascript">
+    window.cookieconsent_options = {{ dict "message" "This website uses cookies." "dismiss" "Got it!" "learnMore" "More info" "link" (.Site.Params.cookie_consent_info_url | absURL)  "theme" "dark-top" | jsonify | safeJS }};
+</script>
+
+<script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/1.0.10/cookieconsent.min.js"></script>
```

Comment:
I think the default should be something unobtrusive - I like my choices because they get noticed as soon as someone looks for the menu (the cookie consent window hides it).

Maybe three variants configurable by one parameter:   simple/elegant/unobtrusive   -  a bit more aggressive - screaming.  Then people have choice but are not required to do all decisions.

Regards, Frank